src/libs/costmap_2d/plugins/static_layer.cpp

```diff
@@ -55,72 +55,91 @@ StaticLayer::StaticLayer() : dsrv_(NULL) {}
 
 StaticLayer::~StaticLayer()
 {
-  if (dsrv_) {
+  /* if (dsrv_) {
     delete dsrv_;
-  }
+  } */
 }
 
 void StaticLayer::onInitialize()
 {
-  ros::NodeHandle nh("~/" + name_), g_nh;
+  auto nh = rclcpp::Node::make_shared(name_);
+  rclcpp::Node::SharedPtr g_nh;
+  auto parameters_client = std::make_shared<rclcpp::SyncParametersClient>(nh);
```

Comment:
You can just use the parameter_client built into the node.

Reply:
I made an issue for this in #119. I was planning to make this replacement all in one PR.